tests/Subscriber/AutoUpdateTitleWithLabelSubscriberTest.php

```diff
@@ -88,42 +82,14 @@ public function testOnPullRequestLabeledCaseInsensitive()
 
     public function testOnPullRequestLabeledWithExisting()
     {
-        $event = new GitHubEvent([
-            'action' => 'labeled',
-            'number' => 1234,
-            'pull_request' => [
+        $event = new GitHubEvent(['action' => 'labeled', 'number' => 1234, 'pull_request' => []], $this->repository);
+        $this->pullRequestApi->method('show')->willReturn([
                 'title' => '[Messenger] Fix JSON',
                 'labels' => [
                     ['name' => 'Status: Needs Review', 'color' => 'abcabc'],
                     ['name' => 'Messenger', 'color' => 'dddddd'],
                 ],
-            ],
-        ], $this->repository);
-
-        $this->dispatcher->dispatch($event, GitHubEvents::PULL_REQUEST);
-        $responseData = $event->getResponseData();
-        $this->assertEmpty($responseData);
-    }
-
-    /**
-     * If a user add two labels at the same time. We will get two webhooks simultaneously.
-     * We need to make sure that when we request the Github API it will return the changes
-     * from the first webhook.
-     */
-    public function testOnPullRequestLabeledTwice()
```

Comment:
This test was removed because if synced the PR data (for "title") with the webhook data. We are not using the webhook data anymore.